drivers/modem/quectel-bg9x.c

```diff
@@ -953,6 +953,64 @@ static const struct setup_cmd setup_cmds[] = {
 	SETUP_CMD_NOHANDLE("AT+QICSGP=1,1,\"" MDM_APN "\",\"" MDM_USERNAME "\", \"" MDM_PASSWORD "\",1"),
 };
 
+
+/* Func: modem_pdp_context_open
+ * Desc: This helper function is called from modem_setup, and is
+ * used to open the PDP context. If there is trouble opening the
+ *  PDP context, we try to deactive and reactive MDM_PDP_ACT_RETRY_COUNT times.
+ *  If it fails, we return an error.
+ */
+
+
+static int modem_pdp_context_open(const int contextID)
+{
+
+	int ret;
+	int pdp_act_retry_count = 0;
+	char buf_act[sizeof("AT+QIACT=##")] = {0};
+	char buf_deact[sizeof("AT+QIDEACT=##")] = {0};
+
+	if (contextID < MDM_MIN_PDP_CONTEXT_ID || contextID > MDM_MAX_PDP_CONTEXT_ID) {
+		LOG_ERR("WRONG CONTEXT ID USED : %d ; SHOULD BE BETWEEN %d AND %d",
+				contextID, MDM_MIN_PDP_CONTEXT_ID, MDM_MAX_PDP_CONTEXT_ID);
```

Comment:
Please don't use uppercase **only**. Such messages would reduce readability if they were all uppercase.

Also there is something wrong with indentation here.